packages/grafana-ui/src/components/Button/Button.tsx

```diff
@@ -364,6 +381,8 @@ function getPropertiesForDisabled(theme: GrafanaTheme2, variant: ButtonVariant,
     boxShadow: 'none',
     color: theme.colors.text.disabled,
     transition: 'none',
+    transform: 'none !important',
```

Comment:
I was able to remove this important without any other changes and the active styles do not appear to be getting applied to disabled buttons